upcase_booklist_app/templates/authors.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body>
+        <div class="page-header">
+            <p class="nav-item">
+                <a href="{{ url_for('book_list') }}">Books</a>
```

Comment:
I _think_ you'll eventually want to put the nav items in a `nav` element with an `ol` and `li` elements. See the example here:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav